app/controllers/distributions_controller.rb

```diff
@@ -33,11 +46,32 @@ def leap
   def tumbleweed
     @hide_search_box = true
     @colour = "primary"
+    @text = "black"
     @distro_type = "tumbleweed"
     @yaml_data = YAML.safe_load(ERB.new(File.read("#{Rails.root}/app/data/tumbleweed.yml.erb")).result(binding))
```

Comment:
I think we should use `load_yaml` here and in the newly added actions, there is no need to copy paste the implementation if it does not differ :)

Reply:
Right